woocommerce/woocommerce
# Migrate `@woocommerce/data` import & export stores to TS

### All Submissions:






### Changes proposed in this Pull Request:



Part of #33105 

This PR migrates `import` and `export` stores to TS

### How to test the changes in this Pull Request:

1. Should run `pnpm nx build @woocommerce/data` without errors
3. Should run `pnpm nx test @woocommerce/data` without errors

### Other information:




### FOR PR REVIEWER ONLY:
